pandas/tests/arrays/categorical/test_missing.py

```diff
@@ -148,3 +148,13 @@ def test_use_inf_as_na_outside_context(self, values, expected):
             result = pd.isna(DataFrame(cat))
             expected = DataFrame(expected)
             tm.assert_frame_equal(result, expected)
+
+    def test_compare_unequal(self):
```

Comment:
`test_compare_categorical_with_missing`

Reply:
Thank you. Fixed.
https://github.com/pandas-dev/pandas/pull/36520/commits/3e293cf0d223322d81a5f6ac632f15973b64335e